src/styles/blocks/_catalog-page.scss

```diff
@@ -0,0 +1,16 @@
+.catalog-page {
+  display: flex;
+  justify-content: center;
+
+  &__wrapper {
+    width: 1000px;
```

Comment:
calculate this width using variables, you can create variable for card width and space between cards then this width will be (card width * 4) + (space between * 3)